### PR TITLE
Feature/awf/chart components to lib#283

### LINKED ISSUE #283: Refactor Lab chart components to a separate module
Use same repo created for #282 to host this new component module.

Goal of this task is to move the following components to a new module in an external repo:
- LineChartComponent
- ChartComponent
- All of the dropdown components

And refactor the Lab to properly use the new external dependency.

### PR DESCRIPTION
**Pull request by CloudNiner**
## Overview

Removes all of the components added in azavea/climate-change-components#7 and replaces them with their equivalents from the library.

### Demo

N/A should look and function the same as before.

### Notes

A few miscellaneous commits that cleanup unused modules/code/dependencies.

Will not merge until companion PR is merged and the components lib version can be updated.

## Testing Instructions

See azavea/climate-change-components#7 for testing instructions, they're the same for this PR

## Checklist
- [x] `yarn run serve` clean?
- [x] `yarn run build:prod` clean?
- [x] `yarn run lint` clean?

Closes #283

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,7 +45,6 @@
     "ng2-nouislider": "^1.6.2",
     "ngx-bootstrap": "^1.9.1",
     "ngx-clipboard": "^8.0.4",
-    "ngx-ui-switch": "^1.3.9",
```
Comment (maurizi) [inline package.json]: I didn't see this get brought in to `climate-change-components` as part of https://github.com/azavea/climate-change-components/pull/7, was it unused?

Comment (CloudNiner) [reply package.json]: Correct.

**Comment by CloudNiner** [comment]
Updated with `climate-change-components@0.2.0`. Run `yarn` to update packages and `yarn run serve` should work as before.

**Comment by maurizi** [comment]
:+1: Still good!

**Pull request merged**